scripts/local/lint.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+TELEPORTER_PATH=$(
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  cd ../.. && pwd
+)
+
+cd $TELEPORTER_PATH/contracts/src
+solhint '**/*.sol'
```

Comment:
Is the `.solhint.json` and `.solhintignore` configs used by default here? The CI workflow uses
```
solhint '**/*.sol' --config ./.solhint.json --ignore-path ./.solhintignore
```

Reply:
yup it looks by default in the directory it's getting called, and then towards parents. Updated to match CI